frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/types.ts

```diff
@@ -11,7 +11,16 @@ export type NodeSelectionCallback = (nodeData: BuilderNodeModelData) => void;
 
 // Node Data Models
 export type PipelineRunAfterNodeModelData = {
+  id?: string;
+  width?: number;
+  height?: number;
   selected?: boolean;
+  status?: RunStatus;
+  whenStatus?: string;
```

Comment:
I like types :smirk: 

```suggestion
  whenStatus?: WhenStatus;
```